welcome-to-sourcery.py

```diff
@@ -18,17 +16,125 @@ def refactoring_example(spellbook):
             result.append(spell)
     return result
 
-# Apply the suggestion by taking the quick-fix action:
-#   Put the cursor on the highlighted line and click the lightbulb or use the hotkey
-#   (Ctrl + .) or (Cmd + .), then take the 'Convert for loop...' option.
-#
-# All of Sourcery's suggestions are shown in the Problems pane (Ctrl/Cmd+Shift+M).
+# Hover over the underlined section to see details of the 
+# changes including a diff. 
 
-# Sourcery also provides code metrics for each function to give you insight into
-# code quality - hover over the function definition above to see this report.
+# You can quickly accept Sourcery's changes with the quick
+# fix action. Put your cursor on the highlighted line and 
+# click on the lightbulb. Or use the quick-fix hotket
```

Comment:
Typo: hotket -> hotkey

We could perhaps even highlight the part about the hotkeys.